packages/node/src/handlers.ts

```diff
@@ -135,8 +136,8 @@ function parseRequest(
 }
 
 /** JSDoc */
-export function requestHandler(): (req: Request, res: Response, next: () => void) => void {
-  return function sentryRequestMiddleware(req: Request, _res: Response, next: () => void): void {
+export function requestHandler(): (req: http.ClientRequest, res: http.ServerResponse, next: () => void) => void {
```

Comment:
As a note, I don't exactly know what kind of style for imports you are following but you can import just import `ClientRequest` and `ServerResponse` to reduce verbosity or are you doing this to avoid type collissions in your builds?